### PR TITLE
feat: added intr token and crowdloan model

### PR DESCRIPTION
**Pull request by alexeiZamyatin**
DO NOT MERGE YET. 

Completed but waiting for public announcement before publishing on official docs site.

### CODE REVIEW

File: docs/interlay/crowdloans.md [hunk 1]
```diff
@@ -1,3 +1,114 @@
 # The Interlay Crowdloan
 
-Coming soon
\ No newline at end of file
+Interlay participates in the Polkadot parachain auctions and supports a crowdloan to enable the community to vote.
+To participate in the crowdloan, you need to lock your DOT and vote for Interlay to become a parachain.
+
+Below is a summary of the Interlay crowdloan details, as described in the [token economy whitepaper released by Kintsugi Labs](https://raw.githubusercontent.com/interlay/whitepapers/master/Interlay_Token_Economy.pdf).
```
Comment (nud3l) [inline docs/interlay/crowdloans.md:6]: Is this Interlay Labs or Kintsugi Labs?

File: docs/interlay/tokenomics.md [hunk 2]
```diff
@@ -1,3 +1,96 @@
 # INTR Tokenomics
 
-Coming soon
\ No newline at end of file
+Interlay will be governed by its community from day 1 - via INTR, Interlay’s governance token. 
+
+
+?> _INTR tokens will be distributed to network participants, builders and early backers as airdrops and block-rewards. There will be no public sale or ICO._
+
+Below is a summary of the Interlay token economy, as described in the [token economy whitepaper released by Kintsugi Labs](https://github.com/interlay/whitepapers).
```
Comment (nud3l) [inline docs/interlay/tokenomics.md:8]: Interlay Labs?

File: docs/interlay/tokenomics.md [hunk 3]
```diff
@@ -1,3 +1,96 @@
...
+
+## Key Functions of INTR
+
+The INTR token’s main purposes are:
+
+* **Governance**- INTR holders vote on governance proposals and elect council members to make operational decisions. 
+* **Utility**. Mit-term, Interlay will support transaction fee payments in INTR and other digital assets, in addition to DOT.
```
Comment (nud3l) [inline docs/interlay/tokenomics.md]:
```suggestion
* **Utility**. Mid-term, Interlay will support transaction fee payments in INTR and other digital assets, in addition to DOT. 
```

File: docs/interlay/crowdloans.md [hunk 4]
```diff
@@ -1,3 +1,114 @@
...
+<tbody>
+  <tr>
+    <td>Early bird</td>
+    <td>10% </td>
+    <td>Contribute before start of Auction #6 (23 December)</td>
+  </tr>
+  <tr>
+    <td>Challenges</td>
+    <td>2.5% <br>2.5% </td>
+    <td>Solve Interlay quiz <a href="">(try now &#10148;)</a> <br>Mint interBTC on testnet <a href="https://bridge.interlay.io/">(try now &#10148;)</a> - MUST use <a href="https://wiki.polkadot.network/docs/learn-accounts">same account!</a></td>
```
Comment (nud3l) [inline docs/interlay/crowdloans.md]:
```suggestion
    <td>Solve Interlay quiz <a href="https://interlay.typeform.com/InterlayQuiz">(try now &#10148;)</a> <br>Mint interBTC on testnet <a href="https://bridge.interlay.io/">(try now &#10148;)</a> - MUST use <a href="https://wiki.polkadot.network/docs/learn-accounts">same account!</a></td>
```

File: docs/interlay/crowdloans.md [hunk 5]
```diff
@@ -1,3 +1,114 @@
...
+  </tr>
+  <tr>
+    <td>Referral program</td>
+    <td>5% </td>
+    <td>of base reward per referred DOT</td>
+  </tr>
+  <tr>
+    <td>Kintsugi Legends</td>
+    <td>2.5% </td>
+    <td>If you contributed to Kintsugi crowdloan (MUST use <a href="https://wiki.polkadot.network/docs/learn-accounts">same account!)</td>
```
Comment (nud3l) [inline docs/interlay/crowdloans.md:86]:
```suggestion
    <td>If you contributed to Kintsugi crowdloan - MUST use <a href="https://wiki.polkadot.network/docs/learn-accounts">same account!</td>
```

Nitpick for consistency with the row two above

File: docs/interlay/crowdloans.md [hunk 6]
```diff
@@ -1,3 +1,114 @@
...
+  </tr>
+  <tr>
+    <td><i>To be confirmed:</i><br/>kBTC user bonus</td>
+    <td>tbc</td>
+    <td>Mint kBTC and/or run a Vault with collateral. Will be announced ad-hoc!</td>
+  </tr>
+</tbody>
+</table>
+
+Any INTR not used for the Core, Early, Referral and Success Airdrop will be distributed proportionally among all participants, further increasing the ratio of INTR per locked DOT.
```
Comment (nud3l) [inline docs/interlay/crowdloans.md]:
```suggestion
Any INTR not used for the Early bird, Challenges, Referral, Kintsugi Legends, and Strong supporters airdrop will be distributed proportionally among all participants, further increasing the ratio of INTR per locked DOT.
```

File: docs/interlay/crowdloans.md [hunk 7]
```diff
@@ -1,3 +1,114 @@
...
+
+## Participating in the Crowdloan
+
+The crowdloan is a fully permissionless event - you make your own judgement on if and how you would like to participate.
```
Comment (nud3l) [inline docs/interlay/crowdloans.md]:
```suggestion
The crowdloan is a fully permissionless event - you make your own judgment if and how you would like to participate.
```

File: docs/interlay/crowdloans.md (hunk 1 above)
Comment (alexeiZamyatin) [reply docs/interlay/crowdloans.md:6]: Kintsugi Labs

File: docs/interlay/tokenomics.md (hunk 2 above)
Comment (alexeiZamyatin) [reply docs/interlay/tokenomics.md:8]: No

File: docs/interlay/tokenomics.md [hunk 8]
```diff
@@ -1,3 +1,96 @@
...
+  - 30% as Vault rewards;
+  - 10% as first crowdloan paradrop;
+  - 25% to the on-chain Treasury;
+  - 5% to stake-to-vote staking rewards.
+* 20% to the Interlay team & early backers who funded the development of the protocol - subject to a 48-week cliff followed by 144 weeks of linear vesting.
+* 10% to a Foundation reserve, to be used for funding ecosystem growth and future development.
+
+Starting in year 5, the community receives 100% of newly minted INTR as part of the 2% annual inflation.
+
+*For a detailed breakdown and explanation of the INTR distribution, check out the [token economy whitepaper released by Interlay Labs](https://github.com/interlay/whitepapers)*
```
Comment (nud3l) [inline docs/interlay/tokenomics.md]:
```suggestion
*For a detailed breakdown and explanation of the INTR distribution, check out the [token economy whitepaper released by Kintsugi Labs](https://github.com/interlay/whitepapers)*
```

**Review by nud3l — APPROVED** [review]

**Pull request merged**